src/DescriptorsCommand.cc

```diff
@@ -934,6 +968,15 @@ Json::Value FindDescriptor::construct_responses(
         }
     }
 
+    if (findDesc.isMember("entities")) {
+        for (auto& ent : findDesc["entities"]) {
+            if (ent.getMemberNames().size() == 0) {
+                findDesc.removeMember("entities");
```

Comment:
I think for standard return, you should leave entities with a blank [] like it returns when FindEntity has no matching members. It gets very difficult to do standard parsing of results when the behavior is so different across scenarios.

Reply:
Not sure, given that re always return a "returned" indicating the number of elements present, and can always be checked for > 0, not to mention that any parse will tell you whether the entities exist or not. For now we keep it consistent with the rest of the commands (they all return no entities block if no property is returned).

Reply:
[
    {
        "FindEntity": {
            "entities": [],
            "returned": 0,
            "status": 0
        }
    }
]

Reply:
I see the confusion. When this type of response was fixed for FindEntity, it was inconsistent with the rest (FindImage, FindVideos, etc). 

For now we keep it consistent with FindImage/FindVideo (they all return no entities block if no property is returned), and we later change them all to match FindEntity of the other way around.